### ISSUE TITLE
Accessibility review

### ISSUE THREAD

**Issue by Whersi** [ministryofjustice/moj-frontend#761]
## What
An Accessibility Specialist will be part of the building block improvement work. But, a final review the building block against WCAG 2.2 criteria is to be completed.

## Why
As part of the MoJ Design System standard, all building blocks within the design system must be WCAG 2.2. compliant.

## Who needs to be involved
- Accessibility Specialist
- Interaction Designer
- Content Designer
- Developer

## Done
When an MoJ Accessibility Specialist confirms the building block is WCAG 2.2 compliant.

## Tasklist

- [x] Meeting with Ben on 1 Oct
- [x] Ben looking through guidance
- [x] Accessibility review of component once built 
- [x] Accessibility review of documentation once finished 
- [x] Content changes made
- [x] IxD changes made
- [x] Dev changes made

**Comment by benproctor-rogers** [comment]
### Initial lightweight review of current Button menu component

- Grey button - colour contrast of hover state does not meet the 3.1 minimum requirement when compared to non hovered state. This only becomes relevant when there are multiple copies of the element in question next to each other. The button menu toggle button on it's own would not trigger a fail, however, when the button is toggled open it does as there are multiple buttons (or links) adjacent to each other in the menu.
- Open button menu - ensure that the focus state also meets contrast standards. The current yellow box does not.
- Whether the button menu opens a series of links or buttons needs to be decided and these need to be marked up appropriately.
- Guidance around using non-redundant button/link text needs to be clear.
- Guidance needs to be given around button menu placement to avoid it being "lost" on the right hand side of the screen for screen magnifier users.

### Next steps

- Talk to Chris about button hover and focus state colours.
- Talk to whole team about button Vs link.
- Review guidance on button text and placement with Helen.

**Comment by robertjmccarthy** [comment]
Ben has completed an [accessibility report](https://docs.google.com/spreadsheets/d/1nYolSfRk5Hz2iYwYTEN3Dks2bh04hI0j02OOWIq-gnQ/edit?gid=0#gid=0) and shared with the team.
